external-crates/move/crates/move-compiler/src/typing/visitor.rs

```diff
@@ -195,11 +398,31 @@ pub trait TypingVisitorContext {
         if self.visit_exp_custom(exp) {
             return;
         }
+        if Self::VISIT_TYPES {
+            self.visit_type(&mut exp.ty);
+        }
         let sp!(_, uexp) = &mut exp.exp;
         match uexp {
             E::ModuleCall(c) => self.visit_exp(&mut c.arguments),
```

Comment:
I think we need to visit the type arguments here?

Reply:
And the parameter types! Thanks for catching this.